jjb/repos/aether-stratum-bfrt.yaml

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: 2020 Open Networking Foundation <info@opennetworking.org>
+#
+# SPDX-License-Identifier: LicenseRef-ONF-Member-Only-1.0
+
+- project:
+    name: aether-stratum-bfrt
+    project: '{name}'
+    
+    jobs:
+        - 'aether-stratum-bfrt-build-nightly':
+            build-timeout: 30
+            target: 'bfrt'
```

Comment:
`target` is passed as a job template parameter, but the value `bfrt` is also hardcoded in `aether-stratum-bfrt-build-nightly` - do we want to have a configurable job template or one specific to `bfrt`?

Reply:
I'll remove the target parameter from here.`aether-stratum-bfrt-build-nightly` is specifically used to build stratum-bfrt images. 
I'll however look into using `aether-stratum-bfrt-build-nightly` as template for nightly and weekly builds. In that case, I will rename the template to `aether-stratum-bfrt-build`